### PR TITLE
Improve init options documentation

### PR DESCRIPTION
**Pull request by lemieux**
@jpjoyal @juliangarritano 
@dannytranlx

### CODE REVIEW

**Comment by juliangarritano** [comment]
I guess we don't want to add the customText stuff yet?

**Comment by juliangarritano** [comment]
Otherwise looks good :+1:

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Initialize the plugin using this code snippet
 </script>
 ```
 
-### In Node.js and Browserify
```
Comment (juliangarritano) [inline README.md]: lol SK in Node.js

**Comment by mspensieri** [comment]
:+1:

**Comment by dannytranlx** [comment]
:+1:

**Pull request merged**